deploy/hub/clusterrole.yaml

```diff
@@ -18,6 +18,10 @@ rules:
 - apiGroups: ["rbac.authorization.k8s.io"]
   resources: ["clusterroles", "clusterrolebindings", "roles", "rolebindings"]
   verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+# Allow hub to create subjectaccessreviews
+- apiGroups: ["authorization.k8s.io"]
+  resources: ["subjectaccessreviews"]
```

Comment:
given a separate process, please use a separate service account and distinct clusterrole

Reply:
separated